### PR TITLE
[Admin] (CATS) Resolve issues - February 2019

### PR DESCRIPTION
**Pull request by kbrandl**

### CODE REVIEW

**Comment by o365devx** [comment]
OPS Build status updates of commit _[866b648](https://github.com/OfficeDev/office-js-docs-pr/pull/749/commits/866b6483769e038dd0addf18c545955cc8f66d2c)_: 

:clock10: **Preparing**: average preparing time is _44 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[866b648](https://github.com/OfficeDev/office-js-docs-pr/pull/749/commits/866b6483769e038dd0addf18c545955cc8f66d2c)_: 

:clock10: **Incremental building**: average incremental building time is _29 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[866b648](https://github.com/OfficeDev/office-js-docs-pr/pull/749/commits/866b6483769e038dd0addf18c545955cc8f66d2c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/loading-the-dom-and-runtime-environment.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-CATS-issues/docs/develop/loading-the-dom-and-runtime-environment.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-authentication.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-CATS-issues/docs/excel/custom-functions-authentication.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-workbooks.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-CATS-issues/docs/excel/excel-add-ins-workbooks.md) | :white_check_mark:Succeeded | |
[docs/onenote/onenote-add-ins-page-content.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-CATS-issues/docs/onenote/onenote-add-ins-page-content.md) | :white_check_mark:Succeeded | |
[docs/reference/overview/onenote-add-ins-javascript-reference.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-CATS-issues/docs/reference/overview/onenote-add-ins-javascript-reference.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C13%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201902132254303877-749%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=OdD7AxM9PLEAbvZ2mgaG2nqMKN2EIYVZ4AhSMMhUy%2FI%3D&st=2019-02-13T22%3A50%3A45Z&se=2019-03-16T22%3A55%3A45Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/develop/loading-the-dom-and-runtime-environment.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ The following events occur when an Outlook add-in starts:
...
 
-One way to check that both the DOM and the runtime environment have finished loading is to use the jQuery [.ready()](http://api.jquery.com/ready/) function: `$(document).ready()`. For example, the following **onReady** event handler makes sure the DOM is first loaded before the code specific to initializing the add-in runs. Subsequently, the **onReady** handler proceeds to use the [mailbox.item](https://docs.microsoft.com/javascript/api/outlook/office.mailbox?view=office-js) property to obtain the currently selected item in Outlook, and calls the main function of the add-in, `initDialer`.
+One way to check that both the DOM and the runtime environment have finished loading is to use the jQuery [.ready()](https://api.jquery.com/ready/) function: `$(document).ready()`. For example, the following **onReady** event handler makes sure the DOM is first loaded before the code specific to initializing the add-in runs. Subsequently, the **onReady** handler proceeds to use the [mailbox.item](https://docs.microsoft.com/javascript/api/outlook/office.mailbox?view=office-js) property to obtain the currently selected item in Outlook, and calls the main function of the add-in, `initDialer`.
```
Comment (AlexJerabek) [inline docs/develop/loading-the-dom-and-runtime-environment.md]: Can we remove the "?view=office-js" from the last link?

**Comment by o365devx** [comment]
OPS Build status updates of commit _[f56d141](https://github.com/OfficeDev/office-js-docs-pr/pull/749/commits/f56d1416596b38b96cb9e48ac563f6035ca53f84)_: 

:clock10: **Preparing**: average preparing time is _44 sec(s)_

**Comment by kbrandl** [comment]
@AlexJerabek thanks for the feedback; I made the change you suggested.

**Comment by o365devx** [comment]
OPS Build status updates of commit _[f56d141](https://github.com/OfficeDev/office-js-docs-pr/pull/749/commits/f56d1416596b38b96cb9e48ac563f6035ca53f84)_: 

:clock10: **Incremental building**: average incremental building time is _29 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[f56d141](https://github.com/OfficeDev/office-js-docs-pr/pull/749/commits/f56d1416596b38b96cb9e48ac563f6035ca53f84)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/loading-the-dom-and-runtime-environment.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-CATS-issues/docs/develop/loading-the-dom-and-runtime-environment.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-authentication.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-CATS-issues/docs/excel/custom-functions-authentication.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-workbooks.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-CATS-issues/docs/excel/excel-add-ins-workbooks.md) | :white_check_mark:Succeeded | |
[docs/onenote/onenote-add-ins-page-content.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-CATS-issues/docs/onenote/onenote-add-ins-page-content.md) | :white_check_mark:Succeeded | |
[docs/reference/overview/onenote-add-ins-javascript-reference.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-fix-CATS-issues/docs/reference/overview/onenote-add-ins-javascript-reference.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C13%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201902132332059898-749%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=yE29a5UROUtAxhAxtNRgc6GST6Z3Q6ytwDx1OQkCmNw%3D&st=2019-02-13T23%3A28%3A21Z&se=2019-03-16T23%3A33%3A21Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**